src/Effects.js

```diff
@@ -1445,6 +1449,67 @@ module.exports = class Effects {
         }
       }
     };
+
+    this.emojis = {
+      emojiList : [],
+      emojiTypes : [],
+
+      init : function () {
+        this.imageLovestruck = p5.createGraphics(100, 100);
+        this.imageLovestruck.scale(3);
+        drawLovestruck(this.imageLovestruck.drawingContext);
+        this.emojiTypes.push(this.imageLovestruck);
+
+        this.imageSmiley = p5.createGraphics(100, 100);
+        this.imageSmiley.scale(3);
+        drawSmiley(this.imageSmiley.drawingContext);
+        this.emojiTypes.push(this.imageSmiley);
+
+        this.imageStarstruck = p5.createGraphics(100, 100);
+        this.imageStarstruck.scale(3);
+        drawStarstruck(this.imageStarstruck.drawingContext);
+        this.emojiTypes.push(this.imageStarstruck);
+
+        this.imageTickled = p5.createGraphics(100, 100);
+        this.imageTickled.scale(3);
+        drawTickled(this.imageTickled.drawingContext);
+        this.emojiTypes.push(this.imageTickled);
+
+        this.imageWink = p5.createGraphics(100, 100);
+        this.imageWink.scale(3);
+        drawWink(this.imageWink.drawingContext);
+        this.emojiTypes.push(this.imageWink);
+      },
+
+      draw : function () {
+        if (p5.frameCount % 10 === 0) {
```

Comment:
I think it would be useful to add a comment here explaining that a new emoji is generated every 10 frames. This will improve the readability of the code and make it easy to update in case we want to make changes in the future.